docs/control.rst

```diff
@@ -380,3 +398,159 @@ Advanced Configuration
+
+If all the above tests fail then a final test is made for a pipeline with a source of *any* and a destination of *any*. If no match occurs then the request is processed without passing through any filters.
+
+If a request is processed by a script in the control dispatcher then this request may pass through multiple filters, one from the source to the script and then one for each script step that performs a set point write or operation. Each of these may be a different pipeline.
+
+Pipeline Execution Models
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When a pipeline is defined it may be set to use a *Shared* execution model or an *Exclusive* execution model. This is only important if any of the filters in the pipeline persist state that impacts future processing.
+
+In a *Shared* execution model one pipeline instance will be created and any requests that use resolve to the pipeline will share the same instance of the pipeline. This saves creating multiple objects within the control dispatcher and is the preferred model to use.
+
+However if the filters in the pipeline store previous data and use it to influence future decisions, such as the |DeltaFilter| this behavior is undesirable as requests from different sources or destined for different destinations may interfere with each other. In this case the *Exclusive* execution model should be used.
+
+In an *Exclusive* execution model a new instance of the pipeline will be created for each distinct source and destination of the control request that utilises the pipeline. This ensures that the different instances of the pipeline can not interfere with each other.
+
+Control Pipeline Management
+---------------------------
+
+The Fledge Graphical User Interface provides a mechanism to manage the control pipelines, this is found in the Control sub menu under the pipelines item.
+
++-----------------+
+| |pipeline_menu| |
++-----------------+
+
+The user is presented with a list of the pipelines that have been created to date and an option in the top right corner to add a new pipeline.
+
++-----------------+
+| |pipeline_list| |
++-----------------+
+
+The list displays the name of the pipeline, the source and destination of the pipeline, the filters in the pipeline, the execution model and the enabled/disabled state of the pipeline.
+
+The user has a number of actions that may be taken from this screen.
+
+  - Enable or disable the pipeline by clicking on the checkbox to the left of the pipeline name
+
+  - Click on the name of the plugin to view and edit the pipeline.
+
+  - Click on the three vertical dots to view the content menu.
+
+    +-------------------------+
+    | |pipeline_context_menu| |
```

Comment:
Undefined substitution referenced.
It should define above in images link section.

Reply:
Added definition of image